README.md

```diff
@@ -36,7 +36,11 @@ Download the pre-compiled package for your operating system.  Install required d
    sudo dpkg -i minc-toolkit<version>.deb
    ```
 
-  Then source the MINC toolkit environment by running (for bash) `source /opt/minc/minc-toolkit-config.sh` or (tcsh) `source /opt/minc/minc-toolkit-config.csh`.
+  Then source the MINC toolkit environment by running (for bash) 
+  `source $mincToolsDirectory/minc-toolkit-config.sh` or (tcsh) 
+  `source $mincToolsDirectory/minc-toolkit-config.csh`,
+  
+  where `$mincToolsDirectory` is the path where the MINC toolkit is installed (e.g. `/opt/minc/` or `opt/minc/1.9.15/` for more recent installs)
```

Comment:
- missing / in front of opt of the version path
- instead of a version number, shouldn't it say /opt/minc/VERSION/ instead of the version number?

Reply:
agree with both.
I thought about the second point, but was hesitant as I am not sure the Minc Toolkit install directory will always be the version number (there are 3 different paths so far depending on whether it is before 1.0.8, between 1.0.8 and 1.0.9 or after 1.0.9:))
but I think your proposal is the most generic.